README.dev.rst

```diff
@@ -0,0 +1,198 @@
+``fairtally`` developer documentation
+=====================================
+
+If you're looking for user documentation, go `here <README.rst>`_.
+
+
+The project setup is documented in `a separate document <project_setup.rst>`_. Feel free to remove this document (and/or the link to this document) if you don't need it.
+
+
+
+Development install
+-------------------
+
+.. code:: shell
+
+    # Create a virtualenv, e.g. with
+    python3 -m venv venv3
+
+    # activate virtualenv
+    source venv3/bin/activate
+
+    # make sure to have a recent version of pip
+    pip install --upgrade pip
+
+    # (from the project root directory)
+    # install fairtally as an editable package
+    pip install --no-cache-dir --editable .
+    # install development dependencies
+    pip install --no-cache-dir --editable .[dev]
+
+Afterwards check that the install directory was added to the ``PATH``
+environment variable. You should then be able to call the executable,
+like so:
+
+.. code:: shell
+
+    fairtally --help
+
+
+Running the tests
+-----------------
+
+Running the tests requires an activated virtualenv with the development tools installed.
+
+.. code:: shell
+
+    # unit tests with mocked representations of repository behavior
+    pytest
+    pytest tests/
+
+
+Running linters locally
+-----------------------
+
+Running the linters requires an activated virtualenv with the development tools installed.
+
+.. code:: shell
+
+    # linter
+    prospector
+
+    # recursively check import style for the fairtally module only
+    isort --recursive --check-only fairtally
+
+    # recursively check import style for the fairtally module only and show
+    # any proposed changes as a diff
+    isort --recursive --check-only --diff fairtally
+
+    # recursively fix  import style for the fairtally module only
+    isort --recursive fairtally
+
+.. code:: shell
+
+    # requires activated virtualenv with development tools
+    prospector && isort --recursive --check-only fairtally
+
+You can enable automatic linting with ``prospector`` and ``isort`` on commit like so:
+
+.. code:: shell
+
+    git config --local core.hooksPath .githooks
+
+Versioning
+----------
+
+Bumping the version across all files is done with bump2version, e.g.
+
+.. code:: shell
+
+    bump2version minor
```

Comment:
Does not work for me

```
usage: bumpversion [-h] [--config-file FILE] [--verbose] [--list] [--allow-dirty] [--parse REGEX] [--serialize FORMAT]
                   [--search SEARCH] [--replace REPLACE] [--current-version VERSION] [--no-configured-files] [--dry-run]
                   --new-version VERSION [--commit | --no-commit] [--tag | --no-tag] [--sign-tags | --no-sign-tags]
                   [--tag-name TAG_NAME] [--tag-message TAG_MESSAGE] [--message COMMIT_MSG] [--commit-args COMMIT_ARGS]
                   part [file [file ...]]
bumpversion: error: the following arguments are required: --new-version
```

Reply:
I have `bumpversion                   0.6.0`

Reply:
We don't have a bump2version setup. I created an issue.